common/src/main/java/com/microsoft/identity/common/internal/migration/TokenCacheItemMigrationAdapter.java

```diff
@@ -85,6 +94,45 @@
         return result;
     }
 
+    public static boolean renewFociTokens(final String clientId,
```

Comment:
This method name is misleading.... it's _testing_ whether or not a token can be renewed, not _actually_ renewing them

Reply:
Thank you, Brian. Addressed.